libr/parse/p/parse_x86_pseudo.c

```diff
@@ -15,86 +15,88 @@
 //    0x0001f41c      eabe76de12       jmp word 0x12de:0x76be [2]
 //    0x0001f56a      ea7ed73cd3       jmp word 0xd33c:0xd77e [6]
 static int replace(int argc, const char *argv[], char *newstr) {
-	int i,j,k;
+	const int kMaxOps = 10;
+	int i,j,k,d;
 	struct {
 		char *op;
 		char *str;
+		int args[kMaxOps];  // XXX can't use flex arrays, all unused will be 0
```

Comment:
This will fail on msvc. Const int is considered varsize arrays. Use a define for kmaxops